scripts/migrations/007-experimental-false.js

```diff
@@ -0,0 +1,106 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+const fs = require('fs');
+const path = require('path');
+
+const { walk } = require('../../utils');
+
+/**
+ * @param {object} bcd Parsed BCD object to be updated in place.
+ */
+const fixExperimental = bcd => {
+  for (const { path, compat } of walk(undefined, bcd)) {
+    if (!compat?.status?.experimental) {
+      continue;
+    }
+
+    // This entry is marked as experimental. Check which browsers support it.
+
+    const supportedIn = new Set();
+
+    for (const [browser, support] of Object.entries(compat.support)) {
+      // Consider only the first part of an array statement.
+      const statement = Array.isArray(support) ? support[0] : support;
+      // Ignore anything behind flag, prefix or alternative name
+      if (statement.flags || statement.prefix || statement.alternative_name) {
+        continue;
+      }
+      if (statement.version_added && !statement.version_removed) {
+        supportedIn.add(browser);
+      }
+    }
+
+    const widelySupported = ['chrome', 'firefox', 'safari'].every(browser =>
```

Comment:
What about mobile browsers?  If a mobile-only feature is supported in both Chrome Android and Safari iOS, shouldn't it also be non-experimental?

Reply:
My follow-up does this.